.travis.yml

```diff
@@ -0,0 +1,28 @@
+language: python
+python: 2.7
```

Comment:
No, it depends only on the time of first travis invocation of the repository, before Jan 1st 2015 its the legacy with sudo as default, for newer repositories its by default container based. So being explicit here is a good idea. See http://docs.travis-ci.com/user/workers/container-based-infrastructure/#Routing-your-build-to-container-based-infrastructure